### ISSUE TITLE
Display a warning message to backup config file after making a vault with gocryptfs

### ISSUE THREAD

**Issue by frnmz** [mpobaschnig/Vaults#27]
gocryptfs in their documentation mention:
`The old principle still applies: Important data should have a backup. Also, keep a copy of your master key (printed on mount) in a safe place. This allows you to access the data even if the gocryptfs.conf config file is damaged or you lose the password.
`

That's why I think it is important to warn user how important `gocryptfs.conf` is and tell them to back it up.
So that when bitrot happens on the config files or user accidentally delete it they can use the backed up conf.

Also I think having an unencrypted version of file as a backup is defeating the purpose of file encryption.

**Describe the solution you'd like**
Display a warning message after making a vault

**Comment by mpobaschnig** [comment]
I don't think we should display this message after creating the vault, since this would probably getting annoying after some time. Maybe when starting Vaults for the first time, displaying this might be a good idea.

Regarding master key from the other issue, maybe we should expose it to the user, since it helps to protect against data loss.

**Comment by frnmz** [comment]
> Maybe when starting Vaults for the first time, displaying this might be a good idea.

Yup, I think specifically after creating the first Vault.

> Regarding master key from the other issue, maybe we should expose it to the user, since it helps to protect against data loss.

Maybe put it together with the warning message that we talk above?
